test/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDaoTest.java

```diff
@@ -1,456 +0,0 @@
-import static org.sagebionetworks.bridge.TestConstants.TEST_STUDY_IDENTIFIER;
-
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import javax.annotation.Resource;
-
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
-import org.joda.time.Period;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.sagebionetworks.bridge.BridgeUtils;
-import org.sagebionetworks.bridge.TestUtils;
-import org.sagebionetworks.bridge.exceptions.BridgeServiceException;
-import org.sagebionetworks.bridge.models.ClientInfo;
-import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
-import org.sagebionetworks.bridge.models.schedules.Activity;
-import org.sagebionetworks.bridge.models.schedules.ActivityType;
-import org.sagebionetworks.bridge.models.schedules.Schedule;
-import org.sagebionetworks.bridge.models.schedules.ScheduleContext;
-import org.sagebionetworks.bridge.models.schedules.SchedulePlan;
-import org.sagebionetworks.bridge.models.schedules.ScheduleType;
-import org.sagebionetworks.bridge.models.schedules.SimpleScheduleStrategy;
-import org.sagebionetworks.bridge.models.studies.Study;
-import org.sagebionetworks.bridge.models.schedules.ScheduledActivity;
-import org.sagebionetworks.bridge.models.schedules.ScheduledActivityStatus;
-import org.sagebionetworks.bridge.services.SchedulePlanService;
-
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Maps;
-import com.google.common.collect.Sets;
-
-@ContextConfiguration("classpath:test-context.xml")
-@RunWith(SpringJUnit4ClassRunner.class)
-public class DynamoScheduledActivityDaoTest {
-    
-    private static final DateTimeZone MSK = DateTimeZone.forOffsetHours(3);
-    
-    private static final int SMALL_PAGE_SIZE = 5;
-    private static final int MID_PAGE_SIZE = 20;
-    
-    @Resource
-    DynamoScheduledActivityDao activityDao;
-
-    @Resource
-    SchedulePlanService schedulePlanService;
-    
-    private List<String> schedulePlanGuids = Lists.newArrayList();
-    
-    private String healthCode;
-    
-    private Study study;
-    
-    @Before
-    public void before() {
-        study = new DynamoStudy();
-        study.setIdentifier(TEST_STUDY_IDENTIFIER);
-        study.setTaskIdentifiers(Sets.newHashSet("tapTest"));
-
-        healthCode = BridgeUtils.generateGuid();
-    }
-    
-    @After
-    public void after() {
-        for (String schedulePlanGuid : schedulePlanGuids) {
-            schedulePlanService.deleteSchedulePlanPermanently(TEST_STUDY, schedulePlanGuid);    
-        }
-        activityDao.deleteActivitiesForUser(healthCode);
-    }
-
-    @SuppressWarnings("deprecation")
-    @Test
-    public void getScheduledActivityHistoryV2() throws Exception {
```

Comment:
Is it worth adding this test to integ tests? (With pagination and everything?)

Reply:
In this and other tests being removed here, I added integration tests to test the paging of some APIs that are not being tested in integration tests, including this older API.

I filed a few JIRAs because there are service endpoints that are not in the SDK and so, not tested in the integration tests. BUT I believe at least some of these endpoints were never used in production, they were the result of some rapid iteration between Erin and I to find an API that she found usable. I am going to audit the logs and remove anything that is unused to try and reduce the complexity here.

Reply:
https://github.com/Sage-Bionetworks/BridgeIntegrationTests/pull/350